### PR TITLE
Feature list_auto_scale_vm_groups for list Autoscaling

### PR DESCRIPTION
**Pull request by Nicodemos305**
Add feature list_auto_scale_vm_groups for list autoscaling of lb.

### CODE REVIEW

**Comment by Nicodemos305** [comment]
Pull request of issue #3

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.5.16
```
Comment (arm4b) [inline CHANGES.md]:
```suggestion
## 0.6.0
```

Per https://semver.org/ if new functionality was added minor version should be incremented.
Patch version is for fixes.

**Review by arm4b — CHANGES_REQUESTED** [review]

File: pack.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ keywords:
     - apache-cloudstack
     - apache-acs
     - cloud
-version: 0.5.15
+version: 0.5.16
```
Comment (arm4b) [inline pack.yaml]:
```suggestion
version: 0.6.0
```

File: CHANGES.md (hunk 1 above)
Comment (arm4b) [reply CHANGES.md]: Can you include this change as well?

**Review by arm4b — APPROVED** [review]
LGTM :+1:

**Pull request merged**